pages/events.js

```diff
@@ -93,4 +93,18 @@ function Events() {
 	);
 }
 
+export const getStaticProps = async () => {
+	const events = await getAllEvents();
+	// Attempt to replace new lines with <br/>, doesnt work
+	// const processedEvents = events.map((event) => (
+	// 	{...event, description: <>{event.description.replace(/\n/g, '<br/>')}</>}));
+	// console.log(processedEvents);
+
+	return {
+		props: {
+			events: events,
+		},
+	};
+};
```

Comment:
don't forget the revalidate prop for [ISR](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration)! (maybe make it update every hour or every day?)